nautobot/utilities/api.py

```diff
@@ -19,7 +19,9 @@ def get_serializer_for_model(model, prefix=""):
     # Serializers for Django's auth models are in the users app
     if app_name == "auth":
         app_name = "users"
-    serializer_name = f"nautobot.{app_name}.api.serializers.{prefix}{model_name}Serializer"
+    serializer_name = f"{app_name}.api.serializers.{prefix}{model_name}Serializer"
```

Comment:
I was thinking that we could potentially move this into a function within `nautobot/utilities` as it seems like we may be reusing it more in the future.

Doesn't have to be in this PR, but maybe a separate issue to track?

Reply:
So far (this PR, #357, #303, #293) in each case we've checked for `app_name in settings.PLUGINS` but the logic that we've needed to conditionally execute in each case has been different. Consolidation of reused logic is absolutely something I'm in favor of but so far I'm not seeing what that might look like in this particular case.

Reply:
Good point!